pkg/cli/mirror/manifests.go

```diff
@@ -261,3 +299,16 @@ func WriteCatalogSource(mapping image.TypedImageMapping, dir string) error {
 	logrus.Infof("Wrote CatalogSource manifests to %s", dir)
 	return nil
 }
+
+// WriteUpdateService will generate an UpdateService object and write it to disk
+func WriteUpdateService(release, graph image.TypedImage, dir string) error {
+	updateService, err := generateUpdateService("update-service", release.Ref, graph.Ref)
+	if err != nil {
+		return err
+	}
+	if err := ioutil.WriteFile(filepath.Join(dir, "updateService.yaml"), updateService, os.ModePerm); err != nil {
```

Comment:
Same principle from above question applies on this local hardcoded filename.

Reply:
This should not be an issue given that we write this into a new result directory every run. Only one `updateService.yaml` would ever be generated per run.